lm_eval/models/gpt2.py

```diff
@@ -119,7 +163,7 @@ def _model_call(self, inps):
         logits returned from the model
         """
         with torch.no_grad():
-            return self.gpt2(inps)[0][:, :, :50257]
+            return self.gpt2(inps)[0][:, :, :self.tokenizer.vocab_size]
```

Comment:
That's a bit annoying but `len(self.tokenizer) != self.tokenizer.vocab_size` and I think you should use the first one. Essentially the difference I observed was when using `additional_special_tokens`. Maybe @SaulLu has more context.

Reply:
@thomasw21 you're right! If you want to get the number of tokens including the `added_tokens` you'll need to do `len(self.tokenizer)`. 

On my side, what I'm wondering is why we needed this hardcoded value before your change.

Reply:
Great! fixed in 0f72a0139647df61464e095adbb50d5c95376f02